clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/security/KubernetesCredentials.java

```diff
@@ -71,7 +72,11 @@ public KubernetesCredentials(KubernetesApiAdaptor apiAdaptor,
 
         HashMap<String, String> secretData = new HashMap<>(1);
         String dockerCfg = String.format("{ \"%s\": { \"auth\": \"%s\", \"email\": \"%s\" } }", account.getV2Endpoint(), account.getBasicAuth(), account.getEmail());
-        dockerCfg = new String(Base64.getEncoder().encode(dockerCfg.getBytes()));
+        try{
```

Comment:
I _think_ the default configuration resulting from using `./gradlew idea`
to generate the idea project artifacts will do what you want.

(If not using IntelliJ, I'm sure there are some other options.)

On Thu, Mar 10, 2016 at 8:05 AM ajaijoy <notifications@github.com> wrote:

> In
> clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/security/KubernetesCredentials.java
> <https://github.com/spinnaker/clouddriver/pull/399#discussion_r55675167>:
>
> > @@ -71,7 +72,11 @@ public KubernetesCredentials(KubernetesApiAdaptor apiAdaptor,
> >
> >          HashMap<String, String> secretData = new HashMap<>(1);
> >          String dockerCfg = String.format("{ \"%s\": { \"auth\": \"%s\", \"email\": \"%s\" } }", account.getV2Endpoint(), account.getBasicAuth(), account.getEmail());
> > -        dockerCfg = new String(Base64.getEncoder().encode(dockerCfg.getBytes()));
> > +        try{
>
> will do, is there a formatter that i can use going forward
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/spinnaker/clouddriver/pull/399/files#r55675167>.
>